### PR TITLE
feat: update responsive text styles to object notation

### PR DESCRIPTION
**Pull request by guilean**

### CODE REVIEW

File: src/texts.ts [hunk 1]
```diff
@@ -88,15 +96,15 @@ export class Texts extends Page {
...
-            ...this.getTextStyles(textStyle),
+            ...this.normalizeTextStyles(textStyle),
             color: 'text-link-accent-default',
           };
         } else if (isMobileText) {
-          this.setResponsiveTextStyles(mobileTextToken, textStyle);
+          this.setResponsiveTextStyles(mobileTextToken, textStyle, 'base');
         } else if (isDesktopText) {
-          this.setResponsiveTextStyles(desktopTextToken, textStyle);
+          this.setResponsiveTextStyles(desktopTextToken, textStyle, 'lg');
```
Comment (tkanzakic) [inline src/texts.ts]: is it possible to only apply this if the value is different that the mobile version?

File: src/texts.ts [hunk 2]
```diff
@@ -12,47 +12,55 @@ export class Texts extends Page {
...
+    }
+    return textStyleWithBreakpoint;
+  };
+
+  addTextStyleBreakpoint = (previousTextStyles, newTextStyles, breakpoint) => {
+    for (let key in previousTextStyles) {
+      previousTextStyles[key][breakpoint] = newTextStyles[key];
     }
+    return previousTextStyles;
   };
```
Comment (tkanzakic) [inline src/texts.ts]: I think this two methods can be easily merged and simply the usage

File: src/texts.ts (hunk 1 above)
Comment (guilean) [reply src/texts.ts]: Added `dedupeResponsiveTextStyles` function to clean duplicated responsive values

File: src/texts.ts (hunk 2 above)
Comment (guilean) [reply src/texts.ts]: Done

**Review by tkanzakic — APPROVED** [review]

**Pull request merged**